plugins/backend-app/readme.md

```diff
@@ -1,26 +1,5 @@
 # @dotcom-tool-kit/backend-app
 
-A bootstrap plugin that provides the minimum required Tool Kit plugins for a "backend" (aka an [API](https://github.com/Financial-Times/next/wiki/Naming-Conventions#apis)). The plugins are:
+## Deprecated
 
-- [`@dotcom-tool-kit/npm`](https://github.com/Financial-Times/dotcom-tool-kit/tree/main/plugins/npm)
-- [`@dotcom-tool-kit/circleci-heroku`](https://github.com/Financial-Times/dotcom-tool-kit/tree/main/plugins/circleci-heroku)
-- [`@dotcom-tool-kit/node`](https://github.com/Financial-Times/dotcom-tool-kit/tree/main/plugins/node)
-- [`@dotcom-tool-kit/husky-npm`](https://github.com/Financial-Times/dotcom-tool-kit/tree/main/plugins/husky-npm)
-- [`@dotcom-tool-kit/secret-squirrel`](https://github.com/Financial-Times/dotcom-tool-kit/tree/main/plugins/secret-squirrel)
-
-This bootstrap plugin is also preconfigured to run the `Node` task on the hook `run:local`.
-
-## Installation
-
-With Tool Kit [already set up](https://github.com/financial-times/dotcom-tool-kit#installing-and-using-tool-kit), install this plugin as a dev dependency:
-
-```sh
-npm install --save-dev @dotcom-tool-kit/backend-app
-```
-
-And add it to your repo's `.toolkitrc.yml`:
-
-```yaml
-plugins:
-    - '@dotcom-tool-kit/backend-app'
-```
+This plugin is deprecated. It is kept to preserve backwards compatibility for projects that previously depended on it, but just imports the `backend-heroku-app`, which supersedes this plugin. 
```

Comment:
**suggestion**: could the deprecated plugins print a warning on `--install`? they could include a dummy hook that just has:

```js
class DeprecatedHook extends Hook {
	check() {
		this.logger.warn(deprecationMessage)
		return true
	}
}
```

would be useful in the future if we decide to deprecate any other plugins

Reply:
**question**: should these plugins be marked as deprecated on npm too?

Reply:
I think we can restrict this to just rely on npm deprecation warnings and documentation as I've found trying to overload the hook installation logic with the logging can be a little awkward (some deprecated plugins don't have any source code atm so would need to add TypeScript config or write OOP in JS; we would need to set unique names for each of the hooks which would then show up in `--help` too; the `check` method runs on every run, not just on `--install`, so there logging would be pretty noisy.) I think the other places highlighting deprecation should be sufficient?

Reply:
I'll add the deprecation messages after this PR is merged!